.binstar.yml

```diff
@@ -1,15 +1,13 @@
 package: poliastro
 install:
-  - conda config --set always_yes true
+  - conda config --set always_yes yes --set changeps1 no
   - conda config --add channels poliastro
 script:
-  # Force conda build to use specific versions of Python and Numpy
-  - export CONDA_PY=$(python -c "from __future__ import print_function;from sys import version_info as ver;print(str(ver.major) + str(ver.minor))")
-  - export CONDA_NPY=$(python -c "from __future__ import print_function;import numpy;npver=numpy.__version__;print(npver[:npver.rfind('.')].replace('.',''))")
-  # Build once the env variables are set
-  - conda build buildscripts/condarecipe
```

Comment:
Well, I think the problem is on what we are trying to do. If a package is meant to be used against a library regardless the version, what should we do? Test it against every single one as posible? And then, shall we repeat the process on any posible combination of the whole bunch of dependencies? It makes sense that in case of no version dependency (or in a range of them), to use the most newer as possible.

But, due to the special status of **NumPy**, I believe they should do an exception (what it seems that they're working on). Hope will see a compromise solution to this issue soon :).